src/script/components/MessagesList/Message/MemberMessage.tsx

```diff
@@ -77,20 +77,17 @@ const MemberMessage: React.FC<MemberMessageProps> = ({
   const isMemberLeave = message.isMemberLeave();
   const isMemberChange = message.isMemberChange();
 
-  const [messageHeaderLabelRef, setMessageHeaderLabelRef] = useEffectRef<HTMLDivElement>();
-  useEffect(() => {
-    if (messageHeaderLabelRef) {
-      const link = messageHeaderLabelRef.querySelector('.message-header-show-more');
-      if (link) {
-        const listener = () => onClickParticipants(highlightedUsers);
-        link.addEventListener('click', listener);
-        return () => {
-          link.removeEventListener('click', listener);
-        };
-      }
+  const initShowMore = useDisposableRef(element => {
```

Comment:
Sadly we still need this `disposableRef` here because the DOM we are manipulating doesn't come from React. It's one of those cases that needs this custom `disposableRef`